src/Modia3D_Interface.jl

```diff
@@ -29,34 +29,34 @@ include("$(ModiaLang.path)/models/Translational.jl")
-Sphere(          ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Sphere)                , kwargs...)
-Ellipsoid(       ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Ellipsoid)             , kwargs...)
-Capsule(         ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Capsule)               , kwargs...)
-Cone(            ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Cone)                  , kwargs...)
+Solid(           ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Solid{FloatType})                 , kwargs...)
+Box(             ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Box{FloatType})                   , kwargs...)
+Beam(            ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Beam{FloatType})                  , kwargs...)
+Cylinder(        ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Cylinder{FloatType})              , kwargs...)
+Sphere(          ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Sphere{FloatType})                , kwargs...)
+Ellipsoid(       ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Ellipsoid{FloatType})             , kwargs...)
+Capsule(         ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Capsule{FloatType})               , kwargs...)
+Cone(            ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Cone{FloatType})                  , kwargs...)
 SpringShape(     ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Spring)                , kwargs...)
 GearWheel(       ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.GearWheel)             , kwargs...)
 Grid(            ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Grid)                  , kwargs...)
 VisualMaterial(  ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.VisualMaterial)        , kwargs...)
-MassProperties(  ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.MassProperties)        , kwargs...)
+MassProperties(  ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.MassProperties{FloatType})        , kwargs...)
 CoordinateSystem(; kwargs...) = Par(; _constructor = :(Modia.Modia3D.CoordinateSystem)      , kwargs...)
 FileMesh(        ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.FileMesh)              , kwargs...)
 Font(            ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Font)                  , kwargs...)
 TextShape(       ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.TextShape)             , kwargs...)
 ModelicaShape(   ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.ModelicaShape)         , kwargs...)
-Fix(             ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Fix)                   , kwargs...)
+Fix(             ; kwargs...) = Par(; _constructor = :(Modia.Modia3D.Fix{FloatType})                   , kwargs...)
 Bushing(         ; kwargs...) = Par(; _constructor = :(Modia3D.Bushing)                     , kwargs...)
 SpringDamperPtP( ; kwargs...) = Par(; _constructor = :(Modia3D.SpringDamperPtP)             , kwargs...)
```

Comment:
`FloatType` is missing for Bushing and SpringDamperPtP. Please sync with: Modia3D/src/ModiaInterface/model3d.jl

Reply:
Thank you. I'll change that.